ur-whitelab/hoomd-tf
# Added RDFs to Run from Trajectory tutorial

This PR fixes the tests and functions that were failing due to doc update and improves the Running from trajectory tutorial to have rdf calculations.

To test this PR:
- Run all tests and make sure they pass (I fixed test_force_matching and test_run_from_traj).
- Open the running from trajectory tutorial notebook and make sure all cells run. You should see two RDFs at the end.
